src/Qowaiv.DomainModel/EventBuffer_TId.cs

```diff
@@ -51,6 +51,9 @@ internal EventBuffer(TId aggregateId, int offset, int committed, AppendOnlyColle
     /// <summary>Gets the committed version of the event buffer.</summary>
     public int CommittedVersion { get; }
 
+    /// <inheritdoc cref="IReadOnlyCollection{T}.Count" />
```

Comment:
Perhaps change the comment of the Count property to indicate explicitly that it should not be confused with the Version property.